### PR TITLE
feat: adding support for exposing tags as xunit category traits #29

### PR DESCRIPTION
**Pull request by videege**
This PR adds support for translating Gherkin tags as "Category" traits on the generated XUnit tests.  This PR also contains some minor typo fixes on class names.

### CODE REVIEW

File: source/Xunit.Gherkin.Quick/Scenario/ScenarioXUnitTestCaseRunner.cs [hunk 1]
```diff
@@ -7,9 +7,9 @@
 
 namespace Xunit.Gherkin.Quick
 {
-    internal sealed class ScenarioXunitTestCaseRunner : XunitTestCaseRunner
+    internal sealed class ScenarioXUnitTestCaseRunner : XunitTestCaseRunner
```
Comment (ttutisani) [inline source/Xunit.Gherkin.Quick/Scenario/ScenarioXUnitTestCaseRunner.cs]: Just to be clear, correct spelling is Xunit, not XUnit. U should be lower case.
Please fix accordingly.

File: source/Xunit.Gherkin.Quick/Scenario/ScenarioDiscoverer.cs [hunk 2]
```diff
@@ -26,9 +26,13 @@ public ScenarioDiscoverer(IMessageSink messageSink)
         {
             var gherkinDocument = GetGherkinDocumentByType(testMethod.TestClass.Class.ToRuntimeType());
 
+            var featureTags = gherkinDocument.Feature.Tags.ToList();
```
Comment (ttutisani) [inline source/Xunit.Gherkin.Quick/Scenario/ScenarioDiscoverer.cs]: So does this mean Feature.Tags cannot be null?

File: source/Xunit.Gherkin.Quick/Scenario/ScenarioXUnitTestCase.cs [hunk 3]
```diff
@@ -25,6 +27,11 @@ public ScenarioXUnitTestCase(IMessageSink messageSink, ITestMethod testMethod, s
                 {  "FeatureTitle", new List<string> { featureName } },
                 {  "Description", new List<string> { scenarioName } }
             };
+
+            if (tags != null && tags.Any())
```
Comment (ttutisani) [inline source/Xunit.Gherkin.Quick/Scenario/ScenarioXUnitTestCase.cs]: Will this ever be null? it seems the caller always passes at least an empty list. Maybe we don't need to check here in that case? And if so, also no need for the default null value in the method signature.

**Comment by ttutisani** [comment]
And how do we run the tests to ensure the tags are accounted? did you run it through command line in a specific way?

**Comment by videege** [comment]
@ttutisani I think I've addressed your comments in my latest commits.

**Comment by ttutisani** [comment]
@videege looks good.
Although, merging is not possible because your branch seems to have conflicts with master branch. You need to fetch master and merge it into your branch. Then push your branch. Then it should be able to build and enable merging.
Thank you!

**Comment by videege** [comment]
OK, I rebased my changes on the latest master branch.

**Pull request merged**